src/extensions.css

```diff
@@ -5,6 +5,7 @@
   /* generated gitako rules */
   /* generated npmhub rules */
   /* generated notifications-preview rules */
+  /* generated sourcegraph rules */
```

Comment:
Since you didnt actually add anything to the generator, this is to be removed. Manual overrides is not generated content.

See https://github.com/StylishThemes/GitHub-Dark#auto-generated-css vs https://github.com/StylishThemes/GitHub-Dark#manual-override-entries

Reply:
I mistakenly thought it would be inherited by the release.